source/MRMesh/MRMeshSubdivide.h

```diff
@@ -28,6 +29,8 @@ struct SubdivideSettings
     const UndirectedEdgeBitSet* notFlippable = nullptr;
     /// New vertices appeared during subdivision will be added here
     VertBitSet * newVerts = nullptr;
+    /// New UV coordinates of new vertices
```

Comment:
if the vector shall be initialized before the algorithm, then it is better to write in the comment